### ISSUE TITLE
Release ? 

### ISSUE THREAD

**Issue by innuendo33** [torchgm/FloatingTB#1]
Hi 

Do you plan to release ? 

Thanks

**Comment by torchgm** [comment]
Yes, I do! Unfortunately right now A) my workstation is out of commission so I can't really work on any of my projects, and B) there's a pretty critical bug that causes Explorer to hang, and makes it very difficult to actually use. I'd rather only release it *after* I've fixed that bug, but hopefully that'll be soon!

**Comment by SapientSi** [comment]
> 是的，我愿意！不幸的是，现在 A） 我的工作站已停运，因此我无法真正处理任何项目，B） 存在一个相当关键的错误，导致资源管理器挂起，并使得它很难实际使用。我宁愿只释放它_后_，我已经修复了该错误，但希望这将是很快！
加油伙计，你是最强的

**Comment by yunus-cakir** [comment]
> 

Ching chang hanji han chon xie do

**Comment by asticassiasuletta** [comment]
> > 
> 
> Ching chang hanji han chon xie do

yea real dignified bud

**Comment by cy4njk** [comment]
hey! any updates on this?

**Comment by SakuraThePythonProgrammer** [comment]
> Hi
> 
> Do you plan to release ?
> 
> Thanks

bruh why is your name innuendo-
